.gitignore

```diff
@@ -1 +1,2 @@
 /node_modules/
+.idea
```

Comment:
Remove this change, please.
I don't want to add IDE specific configuration in this repository.

Reply:
ok